rdg_datasets/five_node_star_graph/README.md

```diff
@@ -0,0 +1,13 @@
+This graph is a star graph containing five nodes.
+It has filler labels and splits that one can use it for ai purposes by adding features.
+add_features scripts is not part of this dataset; one can refer to "add_features.py"
+and "README.md" in "../two_self_loops_ai" to do similar.
+
```

Comment:
you want to show the shape?

Reply:
you mean like the following?:

1  2
^ ^
| /
0
| \
3 4
it did appear nice in the README.md on github.

Reply:
yes, like that

Reply:
I will add this to the REAdME.md then although it might not appear nicely in the github page but will appear nice in the raw file.